ext/standard/url.c

```diff
@@ -190,8 +190,8 @@ PHPAPI php_url *php_url_parse_ex(char const *str, int length)
 			memcpy(port_buf, p, (pp - p));
 			port_buf[pp - p] = '\0';
 			port = strtol(port_buf, NULL, 10);
-			if (port > 0 && port <= 65535) {
-				ret->port = (unsigned short) port;
+			if (port > 0 && port <= 4294967295) {
```

Comment:
> You refer to the port member of the php_url struct and the explicit cast, right?

No, I meant that `port <= 4294967295` will always be true, if long has only 32bits (CMIIW).

> Should be a `zend_long`.

`zend_long` is available as of PHP 7.0 only, but the PR targets 5.6.